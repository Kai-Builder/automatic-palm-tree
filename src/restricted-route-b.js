import React from 'react';
import {Restricted} from './restricted';
import {SecureRoute as Route} from '@okta/okta-react';

export const RestrictedRouteB = (props) => (
  <Route
    path='/restricted-b'
    render={(routeProps) => (
      <Restricted
        {...routeProps}
        {...props}
      />
    )}
  />
);
