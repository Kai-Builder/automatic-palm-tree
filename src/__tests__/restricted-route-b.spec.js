import React from 'react';
import {RestrictedRouteB} from '../restricted-route-b';
import {shallow} from 'enzyme';

describe('RestrictedRouteB', () => {
  const requiredProps = () => ({
  });
  const render = (props = requiredProps()) => shallow(<RestrictedRouteB {...props}/>);

  test('component renders', () => {
    const wrapper = render();

    expect(wrapper).toMatchSnapshot();
  });
});
