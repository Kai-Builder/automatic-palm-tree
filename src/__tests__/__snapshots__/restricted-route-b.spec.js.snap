// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestrictedRouteB component renders 1`] = `
<withAuth(SecureRoute)
  path="/restricted-b"
  render={[Function]}
/>
`;
